tests/CloudFront/SignerTest.php

```diff
@@ -23,22 +23,36 @@ public function setUp()
         );
     }
 
+    /**
+     * Assert that the key is parsed during construction
```

Comment:
Should this doc string description be differentiated from the following test (i.e. variable vs file)?